test/app/user/db/queries_test.clj

```diff
@@ -0,0 +1,222 @@
+(ns app.user.db.queries-test
+  (:require [clojure.test :refer [deftest is testing use-fixtures]]
+            [app.db :as db]
+            [app.db.queries :as db-queries]
+            [app.user.db.queries :as sut]
+            [app.test.fixtures :as fixtures]
+            [db.seed :as dbs])
+  (:import [java.util UUID]))
+
+(use-fixtures :each (fixtures/db db/conn-spec))
+
+(deftest ^:integration add-token-type-test
+  (sut/add-token-type db/conn-spec {:value "id"})
+  (let [token-types (db-queries/get-all-enum-type-values db/conn-spec {:name "token_type"})]
+    (is (= 3 (count token-types)))
+    (is (= #{"access" "refresh" "id"} (set (map :enum_value token-types))))
+    (is (= #{"token_type"} (set (map :enum_name token-types))))))
+
+(deftest ^:integration insert-app-user-test
+  (testing "successful insert"
+    (let [emails    ["test1@example.com" "test2@example.com"]
+          users     (for [email emails]
+                      (sut/insert-app-user db/conn-spec {:email email :password "pass123"}))
+          creations (map :created_at users)]
+      (is (= emails (mapv :email users)))
+      (is (every? nil? (map :deleted_at users)))
+      (is (.before (first creations) (second creations)))))
+
+  (testing "email duplication"
+    (is (thrown-with-msg?
+          org.postgresql.util.PSQLException
+          #"ERROR: duplicate key value violates unique constraint \"app_user_email_key\""
+          (sut/insert-app-user db/conn-spec {:email "test1@example.com" :password "pass123"})))))
+
+(deftest ^:integration batch-insert-app-user-test
+  (testing "successful insert"
+    (let [users   [["batchtest1@example.com" "pass123"]
+                   ["batchtest2@example.com" "pass123"]
+                   ["batchtest3@example.com" "pass123"]]
+          response (sut/batch-insert-app-user db/conn-spec {:users users})
```

Comment:
`sut/batch-insert-app-user` returns map of inserted users already. The following line can be removed.